pkg/rancher-components/src/mixins/labeled-form-element.js

```diff
@@ -0,0 +1,139 @@
+import $ from 'jquery'
```

Comment:
I want to remove `jQuery` everywhere, but I think it's best to keep for now just so we aren't changing too much at once.